### PR TITLE
Setup for 5.2.0 release

### PR DESCRIPTION
**Pull request by pgjones**
Unless there are objections I'll release tomorrow.

This corrects some silly naming (by me) and prepares a release. There really isn't much in the release, but it is required in Hyper-h2 to support the connect protocol.

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ Release History
 6.0.0dev0
 ---------
 
+5.2.0 (2019-01-18)
+------------------
+
+**Other Changes**
```
Comment (Kriechi) [inline HISTORY.rst]: I think this should say `API Changes (Backward-compatible)`, as in previous releases - IFF it applies (?)

**Comment by Kriechi** [comment]
Please also mention "drop py33 support" and "add py37 support" to the release notes.
Otherwise, this looks fine.

File: HISTORY.rst (hunk 1 above)
Comment (pgjones) [reply HISTORY.rst]: I also wasn't sure this counted as an API change, I guess it is - will change.

**Comment by pgjones** [comment]
Updated

**Pull request merged**